addon/components/nrg-datetime/component.js

```diff
@@ -22,6 +22,17 @@ export default Component.extend(Validation, {
 
   maxDate: null, // maximum date/time that can be selected, dates/times after are disabled
 
+  date: null,
```

Comment:
Changing the `value` property to `date` breaks the interface. Please revert back to changing to `value`